Tools/autotest/arducopter.py

```diff
@@ -1044,6 +1044,38 @@ def test_battery_failsafe(self, timeout=300):
 
         self.progress("All Battery failsafe tests complete")
 
+    # Tests the vibration failsafe
+    def test_vibration_failsafe(self, timeout=200):
+        self.progress("Test vibration failsafe")
```

Comment:
This should be redundant in the text output - we should be emitting a banner already

Reply:
removed text, txs